### ISSUE TITLE
Error message appearing on the screen is not automatically read by screen reader.

### ISSUE THREAD

**Issue by prajwalmanju** [microsoft/fluentui#21821]
### Environment Information
- **Browser Version : Version 98.0.1108.43 (Official build) (64-bit)
- OS versions**: 21H2

### Describe the issue:
Error messages must be described with text and compatibility with assistive technology. 

There are error messages that appear on the page with no programmatic call-to-action. For example, keyboard focus is not moved and there is not a live region. In this case, the errors appear dynamically when the page is submitted. As a result, assistive technology will not automatically announce the presence of error messages.

This may particularly disadvantage users that navigate using assistive technology, such as screen readers. Users may struggle to identify and remediate the errors. The lack of a programmatic call-to-action may result in users missing the errors.

#### Actual behavior:

There are error messages that appear on the page with no programmatic call-to-action. For example, keyboard focus is not moved and there is not a live region. In this case, the errors appear dynamically when the page is submitted. As a result, assistive technology will not automatically announce the presence of error messages.

#### Expected behavior:

Screen reader should automatically announce the presence of error messages.

JSON Sample : 

{
  "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
  "type": "AdaptiveCard",
  "version": "1.3",
  "body": [
    {
      "type": "Input.ChoiceSet",
      "id": "requiredCompactId",
      "style": "compact",
      "isMultiSelect": false,
      "label": "Required Input.ChoiceSet label (compact)",
      "isRequired": true,
      "errorMessage": "This is a required input",
      "placeholder" : "Please make selection",
      "choices": [
        {
          "title": "Option 1",
          "value": "1"
        },
        {
          "title": "Option 2",
          "value": "2"
        }
      ]
    },
    {
      "type": "Input.ChoiceSet",
      "id": "requiredExpandedId",
      "style": "expanded",
      "isMultiSelect": false,
      "label": "Required Input.ChoiceSet label (expanded)",
      "isRequired": true,
      "errorMessage": "This is a required input",
      "choices": [
        {
          "title": "Option 1",
          "value": "1"
        },
        {
          "title": "Option 2",
          "value": "2"
        }
      ]
    },
    {
      "type": "Input.ChoiceSet",
      "id": "requiredMultiselectId",
      "style": "expanded",
      "isMultiSelect": true,
      "label": "Required Input.ChoiceSet label (expanded, multiselect)",
      "isRequired": true,
      "errorMessage": "This is a required input",
      "choices": [
        {
          "title": "Option 1",
          "value": "1"
        },
        {
          "title": "Option 2",
          "value": "2"
        }
      ]
    }
  ],
  "actions": [
    {
      "type": "Action.Submit",
      "title": "OK"
    }
  ]
}
